app/src/main/java/com/axiel7/moelist/data/network/Api.kt

```diff
@@ -187,7 +187,8 @@ class Api(private val client: HttpClient) {
         offset: Int?,
         fields: String?,
     ): Response<List<Character>> = client.get("${MAL_API_URL}anime/$animeId/characters") {
-        parameter("limit", limit)
+//        parameter("limit", limit)
+        parameter("limit", 40)
```

Comment:
don't hardcode the value here, pass it to the `getAnimeCharacters` function

Reply:
ok.